utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hwmon.lua

```diff
@@ -0,0 +1,15 @@
+local fs = require "nixio.fs"
+
+local function scrape()
+  local temp_metric = metric("node_hwmon_temp_celsius", "gauge")
+  for dir in fs.glob("/sys/class/thermal/thermal_zone0/hwmon*") do
```

Comment:
On my Turris Omnia, `/sys/class/hwmon/hwmon*/temp*_input` gives me 8 different sensors
```
# cat /sys/class/hwmon/hwmon*/name
f1072004mdiomii01
f10e4078.thermal
mv88e6xxx100
mv88e6xxx101
mv88e6xxx102
mv88e6xxx103
mv88e6xxx104
ath10k_hwmon
```
Why limit ourselves to 1 ?